test-app/tests/integration/components/bs-accordion-test.gjs

```diff
@@ -80,12 +96,18 @@ module('Integration | Component | bs-accordion', function (hooks) {
   });
 
   test('changing selected item expands this item', async function (assert) {
-    this.set('selected', 1);
-    await render(hbs`<BsAccordion @selected={{this.selected}} as |acc|>
-  <acc.item @value={{1}} @title='TITLE1'>CONTENT1</acc.item>
-  <acc.item @value={{2}} @title='TITLE2'>CONTENT2</acc.item>
-</BsAccordion>`);
-    this.set('selected', 2);
+    const data = new (class {
+      @tracked selected = 1;
+    })();
```

Comment:
Are there any established best practices within the community how to create data objects for tests? Personally I feel this style is hard to read. Explicitly declaring the class before may be easier:

```suggestion
    class TestData {
      @tracked selected = 1;
    };
    
    const data = new TestData();
```

But I'm not much opinionated about it.